services/QuillLMS/app/models/stripe_integration/user.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module StripeIntegration
+  class User < SimpleDelegator
```

Comment:
I hadn't used `SimpleDelegator` before.  This is sweet.

Reply:
![Screenshot 2022-12-19 at 6 01 52 PM](https://user-images.githubusercontent.com/2057805/208544394-5b1e7dbf-aa07-4b11-9ba3-e7f2f93afa4d.png)

I feel like it's a good way of carving up our large user class.